packages/wonder-blocks-banner/src/components/banner-icons.js

```diff
@@ -0,0 +1,18 @@
+// @flow
+import {type IconAsset} from "@khanacademy/wonder-blocks-icon";
+
+export const infoIcon: IconAsset = {
+    medium: "M12 4C7.58172 4 4 7.58172 4 12C4 16.4183 7.58172 20 12 20C16.4183 20 20 16.4183 20 12C20 7.58172 16.4183 4 12 4ZM2 12C2 6.47715 6.47715 2 12 2C17.5228 2 22 6.47715 22 12C22 17.5228 17.5228 22 12 22C6.47715 22 2 17.5228 2 12ZM10.25 11.25C10.25 10.6977 10.6977 10.25 11.25 10.25H12C12.5523 10.25 13 10.6977 13 11.25V15.5315C13.4313 15.6425 13.75 16.034 13.75 16.5C13.75 17.0523 13.3023 17.5 12.75 17.5H12C11.4477 17.5 11 17.0523 11 16.5V12.2185C10.5687 12.1075 10.25 11.716 10.25 11.25ZM12.9375 7.875C12.9375 8.49632 12.4338 9 11.8125 9C11.1912 9 10.6875 8.49632 10.6875 7.875C10.6875 7.25368 11.1912 6.75 11.8125 6.75C12.4338 6.75 12.9375 7.25368 12.9375 7.875Z",
+};
```

Comment:
**question:** this icon looks similar to the "info" icon from `wonder-blocks-icon`.  Is it the same?  Could we reuse that icon here?

Reply:
Ah, I believe that's true. I think I might keep it this way anyway just for the sake of consistency, and because we're switching to Phosphor icons soon and I'm not sure how that will affect our existing icons.

Reply:
It's totally good to keep it this way. One small thing that could be helpful is adding a TODO comment pointing to the JIRA ticket to mention that this will be migrated over to Phosphor.

Here's the ticket to track that: https://khanacademy.atlassian.net/browse/WB-1409

Reply:
Ah yes, I do have a comment about that in `banner.js`, but I can add it to `banner-icons.js` too.

Reply:
yes please! I think it would be helpful to see that this will change in the (near) future.